mw-webapp/src/logic/aboutProjectPage/AboutProjectPage.tsx

```diff
@@ -49,6 +65,7 @@ export const AboutProjectPage = () => {
       <Title
         level={HeadingLevel.h2}
         text="About project page"
+        className={titleStyles}
```

Comment:
i think you can use just ternary operator(or clsx) instead of useState+ useEffect

Reply:
thanks via clsx really better